packages/babel-plugin-proposal-class-properties/test/fixtures/private/static-self-method/options.json

```diff
@@ -1,3 +1,3 @@
 {
-  "plugins": ["proposal-class-properties", "transform-block-scoping"]
+  "plugins": ["proposal-class-properties", "proposal-private-methods", "transform-block-scoping"]
```

Comment:
I'm surprised that we need this change :thinking: But it makes sense given the test input.

Reply:
This file can be reduced to only `private-methods`, the other plugins are already in https://github.com/babel/babel/blob/main/packages/babel-plugin-proposal-class-properties/test/fixtures/private/options.json

Reply:
Don't we replace the `plugins` option in tests, without merging them?